recipe/bld.bat

```diff
@@ -81,10 +91,45 @@ set BLAS=MKL
 set INTEL_MKL_DIR=%LIBRARY_PREFIX%
 
 set "libuv_ROOT=%LIBRARY_PREFIX%"
-set "USE_SYSTEM_SLEEF=OFF"
-:: Note that BUILD_CUSTOM_PROTOBUF=OFF (which would use our protobuf) doesn't work properly as of last testing, and results in
-:: duplicate symbols at link time.
-:: set "BUILD_CUSTOM_PROTOBUF=OFF"
+set "USE_SYSTEM_SLEEF=ON"
+
+:: Use our protobuf
+set "BUILD_CUSTOM_PROTOBUF=OFF"
+set "USE_LITE_PROTO=ON"
+
+:: Here we split the build into two parts.
+:: 
+:: Both the packages libtorch and pytorch use this same build script.
+:: - The output of the libtorch package should just contain the binaries that are 
+::   not related to Python.
+:: - The output of the pytorch package contains everything except for the 
+::   non-python specific binaries.
+::
+:: This ensures that a user can quickly switch between python versions without the
+:: need to redownload all the large CUDA binaries.
+
+if "%PKG_NAME%" == "libtorch" (
+  :: For the main script we just build a wheel for libtorch so that the C++/CUDA
+  :: parts are built. Then they are reused in each python version.
+  set BUILD_LIBTORCH_WHL=1
+  set BUILD_PYTHON_ONLY=0
+
+  %PYTHON% setup.py bdist_wheel
+  %PYTHON% -m pip install --find-links=dist torch_no_python --no-build-isolation --no-deps
+
+  :: building libtorch generates an empty __init__.py file. We don't want to package that, it should come from pytorch.
```

Comment:
seems like there might be some patching/contribution upstream to do? also with the linking problems